client/trino-cli/src/test/resources/logging.properties

```diff
@@ -0,0 +1,15 @@
+handlers= java.util.logging.FileHandler, java.util.logging.ConsoleHandler
+
+.level= ALL
+
+java.util.logging.FileHandler.level = INFO
+java.util.logging.FileHandler.pattern = %t/test-trino-cli-%u.log
```

Comment:
It seems all these files are identical, except for the filename. Maybe instead of logging to /tmp we could log to the target directory of the tested module using a relative path, and share the same logging.properties file for all modules? When ci.yml creates artifacts it preserves the directory structure in the ZIP, so each log file would be separate.

Reply:
Yeah looks like this wasn't a great idea to have multiple `logging.properties` files, it's causing issues as there's a duplicate check. Thanks for the suggestion, I was struggling with that!